docs/design/3_1_3_programmable_logic.tex

```diff
@@ -26,21 +26,41 @@ \subsubsection{Device Selection}
 
-MARLANN is capable of performing 8-bit fixed point signed integer arithmetic, matrix multiplication, and max-pooling. After evaluating other open-source FPGA-based accelerators, MARLANN was selected due to its low FPGA area overhead (approx. 5,000 LCs), efficient pipelined architecture, and comprehensive documentation. The use of a pre-designed accelerator also allows the designers to quickly deploy simple ML models without the considerable hardware design overhead implicit in a full-custom solution.
+At a high level, the YOLOv2 accelerator speeds up ML tasks through several optimization strategies. Most prominently, the YOLOv2 accelerator aggressively unrolls and pipelines implicit loops throughout the neural network\footnote{More information about loop unrolling can be found in Xilinx's \textit{Loop Pipelining and Loop Unrolling} guide\cite{xilinx}}. Other optimizations strategies employed include use of 16-bit fixed point representations, parameter re-arrangement, ping-pong buffering\cite{pingpong}, and communication through parallel data buses. The module itself is written in the C language, and is converted to a hardware representation during system configuration through the use of a high-level synthesis (HLS) tool. 
+
+In addition to the YOLOv2 accelerator, two alternative hardware acceleration units were considered for the PLB: \textit{MARLANN}\cite{marlann} and \textit{FREE-TPU}\cite{freetpu}. 
+
+\textit{MARLANN} is an open-source IP core capable of performing 8-bit fixed point signed integer arithmetic, matrix multiplication, and max-pooling. While MARLANN has a low FPGA area overhead (approx. 5,000 LCs), its relative simplicity would necessitate a large amount of effort in developing glue logic to form a functional ML system. The MARLANN project is also not regularly maintained, leading to further concerns regarding future project maintenance/extensibility.
+
+\textit{FREE-TPU} is a free (MIT licensed) acceleration platform developed by EmbedEEP. While the platform allows for greater flexibility in model selection compared to our selected solution (allowing for more less complex models such as MobileNet-YOLO), the project is no longer maintained. In addition, as the source code is unavailable (the API is made available as a shared library), we are prohibited from making appropriate modifications to better suit our platform (ex. modifying image data structures), leading to unacceptably poor performance (less than 0.25 FPS).
```

Comment:
Is that a typo in second line? More less complex?

Reply:
```suggestion
\textit{FREE-TPU} is a free (MIT licensed) acceleration platform developed by EmbedEEP. While the platform allows for greater flexibility in model selection compared to our selected solution (allowing for less complex models such as MobileNet-YOLO), the project is no longer maintained. In addition, as the source code is unavailable (the API is made available as a shared library), we are prohibited from making appropriate modifications to better suit our platform (ex. modifying image data structures), leading to unacceptably poor performance (less than 0.25 FPS).
```